modules/custom-css/custom-css/js/css-editor.js

```diff
@@ -77,7 +77,7 @@ jQuery( function ( $ ) {
 		e.preventDefault();
 
 		$( '#css-mode-select' ).slideUp();
-		$( '#css-mode-display' ).text( $( 'input[name=add_to_existing_display]:checked' ).val() == 'true' ? 'Add-on' : 'Replacement' ); // jshint ignore:line
```

Comment:
Thinking out loud:
- We [set the text contents](http://api.jquery.com/text/#text2) of the element with ID `css-mode-display`
- based on the checked state of the checkbox named `add_to_existing_display`.

Since the `:checked` is part of the selector of that checkbox, it will come back empty if it's not selected, right? But this PR isn't changing the return value of that, or its `.val()`, and it's certainly not truthy. So I guess we're fine :+1: